src/blog/2023/07/images-in-node-red-dashboards.md

```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
 
 - displaying physical tools which should be used to resolve a problem.
 
+## Prerequisites
+
+Before proceeding, ensure that you have the following custom nodes installed, as they will be used throughout this guide:
+
+- [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) - A Node-RED node to perform basic mathematical calculations.
+- [node-red-contrib-image-output](https://flows.nodered.org/node/node-red-contrib-image-output) - A simple way to preview and examine images in your flows.
+- [node-red-contrib-os](https://flows.nodered.org/node/node-red-contrib-os) - Nodes for obtaining system information like CPU usage.
+- [node-red-contrib-string](https://flows.nodered.org/node/node-red-contrib-string) - A string manipulation node based on the lightweight stringjs library.
+- [@flowfuse/node-red-dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) - A set of dashboard nodes for Node-RED.
+- [node-red-node-base64](https://flows.nodered.org/node/node-red-node-base64) - A Node-RED node to encode and decode data to and from base64.
```

Comment:
This implies the section below needs all of these? I don't think that's true?

Reply:
We only use 3 in the ff section, but the rest are used later in the article.

Reply:
Lets move it to the relevant sections

Reply:
Done !